docs/modin_xgboost.rst

```diff
@@ -8,14 +8,13 @@ interfaces could be changed.
 Install XGBoost on Modin
 ------------------------
 
-Modin comes with all the dependencies except ``xgboost`` package by default.
-Currently, distributed XGBoost on Modin is only supported on the Ray backend, therefore, see
-the :doc:`installation page </installation>` for more information on installing Modin with the Ray backend.
-To install ``xgboost`` package you can use ``pip``:
+Modin comes with all the dependencies by default.
```

Comment:
well, this isn't exactly true - we're not going to get `xgboost` when we run `pip install modin`, are we?..

Reply:
Paragraph was changed. I added `xgboost` package in `ray` target of setup.py. Please, check again

Reply:
I wonder why we're requiring `xgboost` now, seems to be an excessive dependency, plus the fact that you're sneaking it into documentation PR isn't correct IMHO :)

Reply:
It was reverted. `xgboost` package installation was added to pydocstyle check only.